<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/css" href="../../resources/stylesheets/wasmsgxml.css"?>
<TMSSource name="TivoliMessages" tmsVersion="1.0" xml:lang="en">
<!-- CMVC MSG File Name = ws/code/runtime/src/com/ibm/ejs/resources/CONMMessages.nlsprops -->
<!-- DO NOT EDIT THIS FILE - This file was generated by the XML/Html & Property emitter -->
<!-- BEGIN MESSAGES -->
<Message ID="CONM6001W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6001W" varFormat="Java">
    MSG_CONM_6001W=CONM6001W: El sistema no puede expandir la agrupación de conexión; se espera a que haya una conexión libre.
  </MsgText>
  <Explanation>
    No se puede crear una conexión.
  </Explanation>
  <UserResponse>
    Verifique que la base de datos pueda crear tantas conexiones como la agrupación de conexiones.
  </UserResponse>
</Message>
<Message ID="CONM7013W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7013W" varFormat="Java">
    MSG_CONM_7013W=CONM7013W: Debe establecerse el ID de usuario y la contraseña para que funcione la conexión de prueba.
  </MsgText>
  <Explanation>
    Se ha establecido el ID de usuario o la contraseña, pero no ambos, en una petición de conexión de prueba.
  </Explanation>
  <UserResponse>
    Especifique tanto el ID de usuario como la contraseña o no especifique ninguno de ellos
  </UserResponse>
</Message>
<Message ID="CONM1006E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1006E" varFormat="Java">
    MSG_CONM_1006E=CONM1006E: No se puede encontrar el nombre de clase DataSource &quot;{0}&quot; para DataSource ({1}).
  </MsgText>
  <Explanation>
    No se puede encontrar el nombre de clase DataSource que se ha proporcionado.
  </Explanation>
  <UserResponse>
    Verifique el archivo archivador Java (JAR) con el nombre de clase especificado está instalado. Si se accede a DataSource mediante una búsqueda de JNDI (Java Naming and Directory Interface) directa en lugar de mediante una referencia de recurso o un persistente gestionado por contenedor (CMP), añada el archivo JAR a la propiedad del sistema JVM (Java Virtual Machine) ws.ext.dirs en el servidor de aplicaciones.
  </UserResponse>
</Message>
<Message ID="CONM6017E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6017E" varFormat="Java">
    MSG_CONM_6017E=CONM6017E: En el método enlist, la conexión ya está asociada a otra transacción.
  </MsgText>
  <Explanation>
    Esta conexión no puede participar en una transacción si ya está asociada con otra transacción.
  </Explanation>
  <UserResponse>
    No sitúe en la memoria caché una conexión que deba utilizar más de una hebra en varias transacciones simultáneas.
  </UserResponse>
</Message>
<Message ID="CONM6020W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6020W" varFormat="Java">
    MSG_CONM_6020W=CONM6020W: Una conexión ha quedado huérfana y se ha devuelto a la agrupación {0}.
{1}
  </MsgText>
  <Explanation>
    Una conexión que estaba en uso por una aplicación se ha mantenido sin utilizar ni se ha cerrado pasado la duración del valor orphanTimeout.
  </Explanation>
  <UserResponse>
    Verifique que todas las conexiones de la aplicación se cierran después de cada uso.
  </UserResponse>
</Message>
<Message ID="CONM7009W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7009W" varFormat="Java">
    MSG_CONM_7009W=CONM7009W: Se ha utilizado un intento de sólo lectura para actualizar el bean {0}.  Se necesita un intento de actualización para eliminar la pérdida de posibles actualizaciones en el bean {0} cuando se utiliza simultáneamente en distintas transacciones.
  </MsgText>
  <Explanation>
    Una transacción almacena una actualización en un bean de entidad de persistencia gestionada por contenedor (CMP) y el bean se despliega  con un intento de acceso de sólo lectura.  Si otra transacción almacena el mismo bean de entidad CMP, las actualizaciones  de una transacción pueden escribirse encima de las de otra transacción.
  </Explanation>
  <UserResponse>
    La aplicación debe evitar que transacciones simultáneas actualicen un bean de entidad CMP.  Si esta acción no es posible, el bean de entidad CMP se debe volver a desplegar mediante un intento de acceso de actualización  en lugar de un intento de acceso de sólo lectura para mantener la coherencia de los datos.
  </UserResponse>
</Message>
<Message ID="CONM6000W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6000W" varFormat="Java">
    MSG_CONM_6000W=CONM6000W: Un recurso no puede procesar la petición &apos;prepare&apos; de un compromiso de dos fases porque sólo da soporte al protocolo de compromiso de una sola fase. DataSource es: {0}
  </MsgText>
  <Explanation>
    El sistema intenta llamar &apos;prepare&apos; en un recurso de una sola fase.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM1005W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1005W" varFormat="Java">
    MSG_CONM_1005W=CONM1005W: Falta la contraseña para el ID de usuario facilitado &quot;{0}&quot; para DataSource ({1}).
  </MsgText>
  <Explanation>
    Se ha facilitado el ID de usuario de DataSource, pero no la contraseña; se ignora el ID de usuario.
  </Explanation>
  <UserResponse>
    Proporcione la contraseña que corresponda al ID de usuario o elimínelo.
  </UserResponse>
</Message>
<Message ID="CONM6016E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6016E" varFormat="Java">
    MSG_CONM_6016E=CONM6016E: El sistema no puede participar en la transacción. Emitiendo la nueva ({0}).
  </MsgText>
  <Explanation>
    La llamada para participar en la transacción ha devuelto false, o ha generado una excepción.
  </Explanation>
  <UserResponse>
    Examine los archivos de anotaciones cronológicas de WebSphere Application Server y de la base de datos para ver si hay mensajes que puedan indicar la razón por la que el recurso no puede participar.
  </UserResponse>
</Message>
<Message ID="CONM7008W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7008W" varFormat="Java">
    MSG_CONM_7008W=CONM7008W: Se necesita un nivel de aislamiento de lectura repetible o más potente para eliminar la pérdida potencial de actualizaciones en el bean {0} si éstas se producen simultáneamente por parte de distintas transacciones.
  </MsgText>
  <Explanation>
    Una transacción almacena una actualización de un bean de entidad CMP. Si otra transacción que se ejecuta  simultáneamente almacena el mismo bean de entidad de persistencia gestionada por contenedor (CMP), las actualizaciones de una transacción pueden escribirse encima de  las de otra transacción.
  </Explanation>
  <UserResponse>
    Verifique si la aplicación puede garantizar que las transacciones simultáneas nunca actualicen el mismo bean de entidad de persistencia gestionada por contenedor (CMP).  Si esta acción no es posible, el bean de entidad CMP se debe volver a desplegar utilizando la lectura repetible de JDBC (Java Database Connectivity) o  un nivel de aislamiento serializable.  Recuerde volver a desplegar el resto de beans a los que se acceda mediante la transacción para utilizar  el mismo nivel de aislamiento.
  </UserResponse>
</Message>
<Message ID="CONM1004W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1004W" varFormat="Java">
    MSG_CONM_1004W=CONM1004W: Falta el ID de usuario para la contraseña facilitada para DataSource ({0}).
  </MsgText>
  <Explanation>
    Se ha facilitado la contraseña de DataSource, pero no el ID de usuario; se ignora la contraseña.
  </Explanation>
  <UserResponse>
    Proporcione el ID de usuario que corresponda a la contraseña o elimínela.
  </UserResponse>
</Message>
<Message ID="CONM7012E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7012E" varFormat="Java">
    MSG_CONM_7012E=CONM7012E: El gestor de conexiones no puede encontrar el archivo archivador Java (JAR) ni el archivo ZIP para la clase DataSource {0} en la vía de acceso de la biblioteca del proveedor para DataSource {1} ni en la classpath.
  </MsgText>
  <Explanation>
    El código debe cargar la clase para DataSource JDBC (Java Database Connectivity),  pero no se ha encontrado el archivo JAR ni el archivo ZIP para la clase DataSource  en la vía de acceso de la biblioteca del proveedor para DataSource ni en la classpath.
  </Explanation>
  <UserResponse>
    Corrija la instalación de WebSphere Application Server para que se pueda encontrar la clase DataSource  en la vía de acceso de la biblioteca del proveedor o añada el archivo JAR o ZIP de la clase DataSource a la classpath.
  </UserResponse>
</Message>
<Message ID="CONM7011W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7011W" varFormat="Java">
    MSG_CONM_7011W=CONM7011W: El gestor de conexiones no puede encontrar el archivo archivador Java (JAR) ni el archivo ZIP para la clase DataSource {0} en la vía de acceso de la biblioteca del proveedor para DataSource {1}.  El código comprueba la classpath para el archivo JAR.
  </MsgText>
  <Explanation>
    El código debe cargar la clase DataSource JDBC,  pero el archivo JAR o ZIP de la clase DataSource no se ha encontrado  en la vía de acceso de la biblioteca del proveedor de DataSource.
  </Explanation>
  <UserResponse>
    Corrija la instalación de WebSphere Application Server para que se pueda encontrar la clase DataSource  en la vía de acceso de la biblioteca del proveedor o añada el archivo JAR o ZIP de la clase DataSource a la classpath.
  </UserResponse>
</Message>
<Message ID="CONM6015E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6015E" varFormat="Java">
    MSG_CONM_6015E=CONM6015E: El sistema no ha podido cargar XAResourceFactory en el método ({0}) porque no se ha encontrado la clase ({1}). Emitiendo la nueva ({2}).
  </MsgText>
  <Explanation>
    No se ha encontrado una clase necesaria. JVM (Java Virtual Machine) o el cargador de clases no han podido encontrar la clase.
  </Explanation>
  <UserResponse>
    Verifique que la clase existe en la classpath y que el usuario puede acceder a ella.
  </UserResponse>
</Message>
<Message ID="CONM2000E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_2000E" varFormat="Java">
    MSG_CONM_2000E=CONM2000E: Connection Manager no podrá obtener una capa de portabilidad para DataSource ({0}).
  </MsgText>
  <Explanation>
    El sistema no puede correlacionar este controlador con una capa de portabilidad, como la capa de portabilidad genérica.
  </Explanation>
  <UserResponse>
    Verifique que el archivo cm.jar está en la classpath y que se puede acceder al mismo.
  </UserResponse>
</Message>
<Message ID="CONM7007I" severity="I" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7007I" varFormat="Java">
    MSG_CONM_7007I=CONM7007I: Correlacionando la siguiente SQLException, con ErrorCode {0} y SQLState {1}, con una StaleConnectionException: {2}
  </MsgText>
  <Explanation>
    Se está produciendo una correlación entre las excepciones.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM6014I" severity="I" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6014I" varFormat="Java">
    MSG_CONM_6014I=CONM6014I: El sistema ha recibido la excepción ({0}) en el método ({1}) y está emitiendo una nueva excepción ({2}). El rastreo de pilas de excepción original es: {3}
  </MsgText>
  <Explanation>
    Este es un mensaje informativo que indica que se ha recibido un tipo de excepción y se ha emitido otro.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM7010W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7010W" varFormat="Java">
    MSG_CONM_7010W=CONM7010W: Puede producirse un punto muerto al almacenar un bean de entidad de persistencia gestionada por contenedor (CMP) {0} durante el proceso de acceso de varios clientes con la misma clave.  Un método de sólo lectura va seguido de una actualización de la base de datos.  Es posible que sea necesario cambiar un método de sólo lectura de un intento de acceso de lectura por un intento de acceso de actualización para evitar posibles problemas de punto muerto.
  </MsgText>
  <Explanation>
    Una transacción almacena una actualización de un bean de entidad CMP. El bean se despliega  con un intento de acceso de sólo lectura y con una lectura repetible de JDBC (Java Database Connectivity) o un nivel de aislamiento  serializable de JDBC. Si otra transacción almacena el mismo bean de entidad CMP, se puede producir  un punto muerto de base de datos como consecuencia del paso de un bloqueo de lectura y compartición a un bloqueo de lectura exclusivo.
  </Explanation>
  <UserResponse>
    La aplicación debe evitar que transacciones simultáneas actualicen un bean de entidad CMP.  Si esta acción no es posible, el bean de entidad CMP se debe volver a desplegar mediante un intento de acceso de actualización  en lugar de un intento de acceso de sólo lectura.
  </UserResponse>
</Message>
<Message ID="CONM1003E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1003E" varFormat="Java">
    MSG_CONM_1003E=CONM1003E: Se ha producido un error al cifrar la contraseña por omisión: {0}
  </MsgText>
  <Explanation>
    Se ha producido una excepción durante el cifrado de una contraseña.  Los datos de la contraseña están dañados.
  </Explanation>
  <UserResponse>
    Sustituya la contraseña dañada por una contraseña correcta.
  </UserResponse>
</Message>
<Message ID="CONM7006E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7006E" varFormat="Java">
    MSG_CONM_7006E=CONM7006E: Se ha producido una excepción al crear la instancia de clase ({0}) para DataSource ({1}): {2}
  </MsgText>
  <Explanation>
    Se ha producido una excepción al crear la instancia de DataSource especificada.
  </Explanation>
  <UserResponse>
    Examine la excepción que se ha generado para obtener más información.
  </UserResponse>
</Message>
<Message ID="CONM6013E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6013E" varFormat="Java">
    MSG_CONM_6013E=CONM6013E: La conexión tiene tareas pendientes y no puede unirse a una transacción global para DataSource ({0}).
  </MsgText>
  <Explanation>
    La conexión tiene trabajo que no está comprometido o no se ha retrotraído.
  </Explanation>
  <UserResponse>
    Comprometa o retrotraiga el trabajo de la conexión antes de utilizarla en una transacción global.
  </UserResponse>
</Message>
<Message ID="CONM1002E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1002E" varFormat="Java">
    MSG_CONM_1002E=CONM1002E: Se ha producido un error al descifrar la contraseña por omisión: {0}
  </MsgText>
  <Explanation>
    Se ha producido una excepción durante el descifrado de una contraseña.  Puede que los datos de la contraseña estén dañados o que se codificaran con un cifrado no soportado.
  </Explanation>
  <UserResponse>
    Sustituya la contraseña cifrada por datos no dañados o asegúrese de que se codifique con un algoritmo de cifrado con soporte.
  </UserResponse>
</Message>
<Message ID="CONM6009E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6009E" varFormat="Java">
    MSG_CONM_6009E=CONM6009E: La base de datos no puede obtener una conexión con la base de datos de DataSource ({0}).
  </MsgText>
  <Explanation>
    La base de datos no ha podido atender una petición de una nueva conexión.
  </Explanation>
  <UserResponse>
    Examine el código de estado de SQL de la base de datos para determinar la causa exacta del problema.
  </UserResponse>
</Message>
<Message ID="CONM8002E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_8002E" varFormat="Java">
    MSG_CONM_8002E=CONM8002E: El método que se ha invocado no tiene soporte.
  </MsgText>
  <Explanation>
    El método invocado no tiene soporte en este gestor de conexiones compatible con JDBC 2.0.
  </Explanation>
  <UserResponse>
    No invoque este método.
  </UserResponse>
</Message>
<Message ID="CONM6008W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6008W" varFormat="Java">
    MSG_CONM_6008W=CONM6008W: El sistema ha excedido el tiempo de espera al esperar una conexión de DataSource ({2}). Para información acerca de las conexiones activas, establezca la propiedad DataSource &quot;{0}&quot; en {1} en DataSource &quot;{2}&quot;.
  </MsgText>
  <Explanation>
    Todas las conexiones de la agrupación estaban en uso durante el período de tiempo de espera de conexión.
  </Explanation>
  <UserResponse>
    1) Aumente el valor del tiempo de espera de conexión. 2) Aumente el número máximo de conexiones y verifique que la base de datos puede manejar una carga de conexiones superior.
  </UserResponse>
</Message>
<Message ID="CONM7005E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7005E" varFormat="Java">
    MSG_CONM_7005E=CONM7005E: La clase ({0}) no implementa la interfaz javax.sql.ConnectionPoolDataSource ni la interfaz javax.sql.XADataSource.
  </MsgText>
  <Explanation>
    La clase DataSource que se proporciona debe implementar la interfaz javax.sql.ConnectionPoolDataSource o la interfaz javax.sql.XADataSource.
  </Explanation>
  <UserResponse>
    Consulte la documentación de la base de datos y proporcione el nombre de clase que implemente una de estas interfaces.
  </UserResponse>
</Message>
<Message ID="CONM1001E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1001E" varFormat="Java">
    MSG_CONM_1001E=CONM1001E: Falta la propiedad necesaria ({0}) para DataSource ({1}).
  </MsgText>
  <Explanation>
    Falta una propiedad obligatoria.
  </Explanation>
  <UserResponse>
    Verifique que todas las propiedades necesarias están configuradas en DataSource.
  </UserResponse>
</Message>
<Message ID="CONM6012E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6012E" varFormat="Java">
    MSG_CONM_6012E=CONM6012E: Se ha producido un error de la base de datos al establecer el valor AutoCommit para DataSource ({0}).
  </MsgText>
  <Explanation>
    La base de datos no puede atender la petición.
  </Explanation>
  <UserResponse>
    Examine el código de estado de SQL de la base de datos para determinar la causa exacta del problema.
  </UserResponse>
</Message>
<Message ID="CONM7004W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7004W" varFormat="Java">
    MSG_CONM_7004W=CONM7004W: Se ha producido una excepción al establecer la propiedad {0} en la clase {1}: {2}
  </MsgText>
  <Explanation>
    Se ha producido una excepción al configurar la propiedad especificada para DataSource.
  </Explanation>
  <UserResponse>
    Examine la excepción que se ha generado para obtener más información.
  </UserResponse>
</Message>
<Message ID="CONM8001E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_8001E" varFormat="Java">
    MSG_CONM_8001E=CONM8001E: El método {0} no está soportado en una clase DataSource compatible con JDBC (Java Database Connectivity) V2.0.
  </MsgText>
  <Explanation>
    El método invocado es un método compatible con JDBC V4.0 y no está soportado en este gestor de conexiones compatible con JDBC 2.0.
  </Explanation>
  <UserResponse>
    No invoque este método o utilice un mecanismo de agrupación de conexiones compatible con JDBC 4.0.
  </UserResponse>
</Message>
<Message ID="CONM6027W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6027W" varFormat="Java">
    MSG_CONM_6027W=CONM6027W: Una conexión ha quedado huérfana y se ha devuelto a la agrupación {2}.  Para obtener información sobre que vía de acceso de código está causando que las conexiones se queden huérfanas, establezca la propiedad DataSource &quot;{0}&quot; en {1} en DataSource &quot;{2}&quot;.
  </MsgText>
  <Explanation>
    Este mensaje de aviso describe como activar la información de diagnóstico para detectar vías de acceso de código de conexiones huérfanas.
  </Explanation>
  <UserResponse>
    Establezca las opciones de diagnóstico en el valor indicado en el mensaje.
  </UserResponse>
</Message>
<Message ID="CONM1000E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_1000E" varFormat="Java">
    MSG_CONM_1000E=CONM1000E: El valor &quot;{0}&quot; no es válido para la propiedad ({1}) para DataSource ({2}).
  </MsgText>
  <Explanation>
    Una de las propiedades de origen de datos está en un formato no esperado.
  </Explanation>
  <UserResponse>
    Verifique que todas las propiedades del DataSource son correctas.
  </UserResponse>
</Message>
<Message ID="CONM6011E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6011E" varFormat="Java">
    MSG_CONM_6011E=CONM6011E: Se ha producido un error interno de la agrupación de conexiones.  Se ha detectado un estado ConnectO no permitido, &quot;{0}&quot;, en DataSource ({1}).
  </MsgText>
  <Explanation>
    Excepción inesperada. La causa no se puede determinar inmediatamente.
  </Explanation>
  <UserResponse>
    Si el problema persiste, consulte la determinación de problemas en la página Web de soporte de WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="CONM6007I" severity="I" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6007I" varFormat="Java">
    MSG_CONM_6007I=CONM6007I: Se ha destruido la agrupación de conexiones para DataSource ({0}).
  </MsgText>
  <Explanation>
    Este mensaje está previsto durante la conclusión de un servidor de aplicaciones.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM7003W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7003W" varFormat="Java">
    MSG_CONM_7003W=CONM7003W: El sistema no ha encontrado el método para grabar la propiedad {0} en la clase {1}
  </MsgText>
  <Explanation>
    El método para grabar la propiedad no existe en la clase.
  </Explanation>
  <UserResponse>
    La propiedad especificada existe, pero no existe un método de grabación. Verifique que la propiedad especificada se puede grabar en DataSource.
  </UserResponse>
</Message>
<Message ID="CONM8000E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_8000E" varFormat="Java">
    MSG_CONM_8000E=CONM8000E: El método {0} no está soportado en una clase DataSource compatible con JDBC (Java Database Connectivity) V2.0.
  </MsgText>
  <Explanation>
    El método invocado es un método compatible con JDBC V3.0 y no está soportado en este gestor de conexiones compatible con JDBC 2.0.
  </Explanation>
  <UserResponse>
    No invoque este método o utilice un mecanismo de agrupación de conexiones compatible con JDBC 3.0.
  </UserResponse>
</Message>
<Message ID="CONM6010W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6010W" varFormat="Java">
    MSG_CONM_6010W=CONM6010W: Se ha producido un error StaleConnectionException. Depurando toda la agrupación de conexiones para DataSource ({0}).
  </MsgText>
  <Explanation>
    El sistema ha emitido un error de StaleConnectionException.
  </Explanation>
  <UserResponse>
    Examine el código de estado de SQL de la base de datos para determinar la causa exacta del problema.
  </UserResponse>
</Message>
<Message ID="CONM6026W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6026W" varFormat="Java">
    MSG_CONM_6026W=CONM6026W: Tiempo de espera excedido al esperar una conexión de DataSource ({0}).
{1}
  </MsgText>
  <Explanation>
    Este mensaje de aviso muestra todos los rastreadores de conexión activos cuando se produce un tiempo de espera excedido.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM7019E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7019E" varFormat="Java">
    MSG_CONM_7019E=CONM7019E: El gestor de conexiones ha intentado utilizar una DataSource V4.0 desde un servlet con la versión 2.3 o posterior, que no es una configuración válida.
  </MsgText>
  <Explanation>
    Sólo se pueden utilizar servlets con versiones anteriores a la 2.3 con DataSource V4.0.
  </Explanation>
  <UserResponse>
    Repita la configuración para utilizar una versión anterior de servlet, o configure el servlet para utilizar DataSource V5.0.
  </UserResponse>
</Message>
<Message ID="CONM7002W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7002W" varFormat="Java">
    MSG_CONM_7002W=CONM7002W: El sistema no ha podido encontrar la propiedad {0} en la clase {1}
  </MsgText>
  <Explanation>
    La propiedad que se establece no existe en la clase.
  </Explanation>
  <UserResponse>
    Verifique que el nombre de propiedad especificado es correcto.
  </UserResponse>
</Message>
<Message ID="CONM6006E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6006E" varFormat="Java">
    MSG_CONM_6006E=CONM6006E: Falta la contraseña por omisión necesaria para el ID de usuario por omisión facilitado, &quot;{0}&quot;, para DataSource ({1}).
  </MsgText>
  <Explanation>
    Falta la contraseña por omisión.
  </Explanation>
  <UserResponse>
    Proporcione una contraseña válida para el usuario por omisión.
  </UserResponse>
</Message>
<Message ID="CONM7018E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7018E" varFormat="Java">
    MSG_CONM_7018E=CONM7018E: El gestor de conexiones ha intentado utilizar una clase DataSource V4.0 con un enterprise bean que no es válido. El módulo Enterprise JavaBeans (EJB) actual es:  {0}
  </MsgText>
  <Explanation>
    Sólo los enterprise beans de los módulos V1.1 pueden utilizarse con DataSource V4.0.
  </Explanation>
  <UserResponse>
    Vuelva a configurar el enterprise bean en un módulo 1.1 o configure el enterprise bean para que utilice DataSource V5.0.
  </UserResponse>
</Message>
<Message ID="CONM6025I" severity="I" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6025I" varFormat="Java">
    MSG_CONM_6025I=CONM6025I: Las opciones siguientes de diagnóstico del gestor de conexiones están activadas en DataSource {0}: {1}
  </MsgText>
  <Explanation>
    Este es un mensaje informativo que indica que se han establecido las opciones de diagnóstico del gestor de conexiones.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM6024W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6024W" varFormat="Java">
    MSG_CONM_6024W=CONM6024W: Se ha capturado una excepción ({0}) en el método createConnection.
  </MsgText>
  <Explanation>
    Este mensaje de aviso indica que se ha producido una excepción SQL al crear una conexión.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM7001E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7001E" varFormat="Java">
    MSG_CONM_7001E=CONM7001E: Se ha producido la siguiente excepción al efectuar una introspección de la clase {0}: {1}
  </MsgText>
  <Explanation>
    Se ha producido un error durante la introspección de la clase DataSource.
  </Explanation>
  <UserResponse>
    Examine la excepción que se ha generado para obtener más información. Si el problema persiste, consulte la determinación de problemas en la página Web de soporte de WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="CONM7017E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7017E" varFormat="Java">
    MSG_CONM_7017E=CONM7017E: Se ha producido una excepción en delistResource para la transacción local: {0}
  </MsgText>
  <Explanation>
    Se ha recibido dicha excepción durante la exclusión de la transacción local.
  </Explanation>
  <UserResponse>
    Examine la excepción para determinar el origen del problema. Si el problema persiste, consulte la determinación de problemas en la página Web de soporte de WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="CONM7020W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7020W" varFormat="Java">
    MSG_CONM_7020W=CONM7020W: El gestor de conexiones no puede determinar el tipo de base de datos del programa de fondo.  No se utilizan las semánticas de DB2/390.
  </MsgText>
  <Explanation>
    El tiempo de ejecución no puede determinar si el programa de fondo DB2Connect es una base de datos DB2/390.
  </Explanation>
  <UserResponse>
    Si el problema persiste, consulte la determinación de problemas en la página Web de soporte de WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="CONM7000W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7000W" varFormat="Java">
    MSG_CONM_7000W=CONM7000W: Se ha encontrado una base de datos o controlador no reconocido {0}; se utilizan valores genéricos.
  </MsgText>
  <Explanation>
    El ID de producto o el nombre de clase de controlador de la base de datos no se han encontrado en la lista de bases de datos conocidas.
  </Explanation>
  <UserResponse>
    Compruebe que estén bien escritos y que se dé soporte a la base de datos.
  </UserResponse>
</Message>
<Message ID="CONM7016W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7016W" varFormat="Java">
    MSG_CONM_7016W=CONM7016W: Se ha producido un error en el método ConnectO.destroy durante el proceso de exclusión desde la transacción local: {0}
  </MsgText>
  <Explanation>
    La exclusión de la lista es una operación JDBC (Java Database Connectivity) especificada.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM6004E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6004E" varFormat="Java">
    MSG_CONM_6004E=CONM6004E: Falta el ID de usuario por omisión necesario para la contraseña por omisión facilitada para DataSource ({0}).
  </MsgText>
  <Explanation>
    Se ha proporcionado una contraseña, pero no se ha proporcionado un ID de usuario.
  </Explanation>
  <UserResponse>
    Proporcione un ID de usuario y una contraseña válidos, o elimine la contraseña.
  </UserResponse>
</Message>
<Message ID="CONM0000E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_0000E" varFormat="Java">
    MSG_CONM_0000E=CONM0000E: Se han eliminado las interfaces de programas de aplicación del gestor de conexiones. Utilice las interfaces de programas de aplicación de agrupación de conexiones.
  </MsgText>
  <Explanation>
    Los métodos que se llaman se han eliminado.
  </Explanation>
  <UserResponse>
    Utilice la nueva API de agrupación de conexiones.
  </UserResponse>
</Message>
<Message ID="CONM6019W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6019W" varFormat="Java">
    MSG_CONM_6019W=CONM6019W: Se ha producido una SQLException al validar la conexión para DataSource {0}.  La excepción es {1}.
  </MsgText>
  <Explanation>
    Se ha recibido una SQLException al validar la conexión.  Se depura la agrupación de conexiones y se devuelve una conexión nueva desde la agrupación.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM6002W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6002W" varFormat="Java">
    MSG_CONM_6002W=CONM6002W: El sistema no ha podido obtener un recurso XA {0}
  </MsgText>
  <Explanation>
    El sistema no ha podido obtener un recurso XA mientras se destruía la conexión.  Efectuar una llamada a &apos;destroy&apos; significa cerrar la conexión y eliminarla de la agrupación libre.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción del usuario.
  </UserResponse>
</Message>
<Message ID="CONM7015E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7015E" varFormat="Java">
    MSG_CONM_7015E=CONM7015E: No se ha establecido la propiedad was.install.library.
  </MsgText>
  <Explanation>
    Es necesario establecer la propiedad was.install.library.
  </Explanation>
  <UserResponse>
    Establezca la propiedad was.install.library.
  </UserResponse>
</Message>
<Message ID="CONM7014W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_7014W" varFormat="Java">
    MSG_CONM_7014W=CONM7014W: Se ha producido un error en la conexión de prueba con DataSource : {0}
  </MsgText>
  <Explanation>
    Se ha intentado una conexión de prueba desde la consola administrativa.  La operación ha fallado con la excepción especificada.
  </Explanation>
  <UserResponse>
    Examine la excepción para obtener una explicación del error de conexión.
  </UserResponse>
</Message>
<Message ID="CONM6018E" severity="E" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6018E" varFormat="Java">
    MSG_CONM_6018E=CONM6018E: La inclusión de unilateralCommit no recibe soporte en una conexión JTAEnabled en una transacción global.
  </MsgText>
  <Explanation>
    Se ha llamado unilateralCommit en una conexión JTAEnabled de una transacción global.
  </Explanation>
  <UserResponse>
    No utilice unilateralCommit en una conexión JTAEnabled en el código de aplicación. Si el problema persiste, consulte la determinación de problemas en la página Web de soporte de WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="CONM6021W" severity="W" prefix="yes">
  <MsgText pgmKey="MSG_CONM_6021W" varFormat="Java">
    MSG_CONM_6021W=CONM6021W: Una propiedad de proveedor de JDBC (Java Database Connectivity) (TransactionBranchesLooselyCoupled) está establecida en Oracle. Antes de establecer esta propiedad debe aplicarse el parche 2511780 de Oracle.
  </MsgText>
  <Explanation>
    Se ha establecido una propiedad personalizada al crear el proveedor de JDBC en un sistema Oracle. La propiedad requiere el parche 2511780 de Oracle.
  </Explanation>
  <UserResponse>
    Si se instala el parche, no se requiere ninguna acción de usuario. De lo contrario, póngase en contacto con el servicio de soporte de Oracle para obtener el parche.
  </UserResponse>
</Message>
<!-- END MESSAGES -->
</TMSSource>
